metadata-editor/app/controllers/EditsController.scala

```diff
@@ -73,25 +73,22 @@ class EditsController(
   // TODO: Think about calling this `overrides` or something that isn't metadata
   def getAllMetadata(id: String) = auth.async {
     val emptyResponse = respond(Edits.getEmpty)(editsEntity(id))
-    editsStore.get(id) map { dynamoEntry =>
-      dynamoEntry.asOpt[Edits]
+    editsStore.getV2(id) map { record =>
+      record
         .map(respond(_)(editsEntity(id)))
         .getOrElse(emptyResponse)
-    } recover { case NoItemFound => emptyResponse }
+    }
   }
 
   def getEdits(id: String) = auth.async {
-    editsStore.get(id) map { dynamoEntry =>
-      val edits = dynamoEntry.asOpt[Edits]
-      respond(data = edits)
+    editsStore.getV2(id) map { record =>
+      respond(data = record)
     } recover { case NoItemFound => NotFound }
```

Comment:
It looks like you've removed this `recover` block from the other methods nearby, should we remove it here too? I guess `NoItemFound` can no longer be thrown?